MVVM Module/Tests/Feed UI/Helpers/FeedViewController+TestHelpers.swift

```diff
@@ -57,4 +57,16 @@ extension FeedViewController {
 	private var feedImagesSection: Int {
 		return 0
 	}
+
+	var errorView: ErrorView? {
```

Comment:
This helper is not used from outside of this file thus it can be private, please make it private
```suggestion
	private var errorView: ErrorView? {
```

Reply:
Ok!